web/src/stores/clips.ts

```diff
@@ -1,16 +1,71 @@
 import { Action as ReduxAction, Dispatch } from 'redux';
-const contributableLocales = require('../../../locales/contributable.json') as string[];
+const contributableLocales =
+  require('../../../locales/contributable.json') as string[];
 import StateTree from './tree';
 import { User } from './user';
 import { Clip } from 'common';
 
+async function getCanPlayAudio(audioSrc: string) {
+  return new Promise(resolve => {
+    const audio = new Audio(audioSrc);
+
+    audio.addEventListener('error', () => {
+      resolve(false);
```

Comment:
should this be `reject()`

Reply:
It could be for sure, the function does return whether or not it can play audio so I think it erroring is an expected state